sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -96,7 +109,23 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     programOffering.parentOffering = programOffering;
 
     try {
-      return await this.repo.save(programOffering);
+      return await this.dataSource.transaction(
+        async (transactionalEntityManager) => {
+          await this.saveEducationProgramOfferingNotification(
+            educationProgramOffering.offeringName,
+            educationProgramOffering.programContext.name,
+            educationProgramOffering.operatingName,
+            educationProgramOffering.legalOperatingName,
+            educationProgramOffering.primaryEmail,
+            programOffering.offeringStatus,
+            programOffering.institutionLocation.name,
+            transactionalEntityManager,
```

Comment:
Not a blocker, if the arguments for a method is more than 5 it is good to create model to be passed.